server/src/main/java/io/javaclasses/filehub/api/UnauthorizedException.java

```diff
@@ -0,0 +1,17 @@
+package io.javaclasses.filehub.api;
+
+/**
+ * An exception that should be raised in case the user's login credentials are incorrect.
```

Comment:
The last part is not clear.
```suggestion
 * An exception that should be raised when user with login credentials is not found.
```